main.c

```diff
@@ -28,6 +28,7 @@ static const char usage[] =
 	"                                  meaning as in CSS.\n"
 	"      --border-size <px>          Border size.\n"
 	"      --border-color <color>      Border color.\n"
+	"      --progress-color <color>    Progress bar color.\n"
```

Comment:
Should call it the "progress indicator" to be consistent with the long description in the manpage.